src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -1259,12 +1275,54 @@ Don't worry, you'll have the option to save your work.</value>
     <value>New</value>
     <comment>Displays next to the package name in the Package Search window if the package has been added in the last 30 days.</comment>
   </data>
-  <data name="PackageManagerSearchPackagesControlName" xml:space="preserve">
-    <value>packageManagerSearchPackages</value>
-  </data>
   <data name="PackageManagerMyPackagesControlName" xml:space="preserve">
     <value>packageManagerMyPackages</value>
   </data>
+  <data name="PackageManagerSearchTab" xml:space="preserve">
+    <value>Search for Packages</value>
```

Comment:
We used to have `Search for a Package...` I assume all the choices of these new terms have been considered as change @hwahlstrom

Reply:
Leaving it as it is then!